cordformation/src/main/kotlin/net/corda/plugins/Cordformation.kt

```diff
@@ -41,7 +41,8 @@ class Cordformation : Plugin<Project> {
             val releaseVersion = project.findRootProperty<String>("corda_release_version")
                     ?: throw IllegalStateException("Could not find a valid declaration of \"corda_release_version\"")
             val maybeJar = project.configuration("runtime").filter {
-                "$jarName-$releaseVersion.jar" in it.toString() || "$jarName-enterprise-$releaseVersion.jar" in it.toString()
+                it.toString().contains("$jarName-enterprise-$releaseVersion(-*)?.jar".toRegex()) ||
```

Comment:
Compiling a `Regexp` is expensive. Can you factor these out please to ensure that they're only compiled once?

I suspect you want `(-.*)?\\.jar\$` at the endof the "enterprise" one, though.

Reply:
Given the frequency of use of the plugins I hardly think this is a performance issue.

Reply:
The regex's have been tested to work.

Reply:
I suspect the `-enterprise` one _hasn't_ been tested because it says `(-*)`, which means "0 or more occurrences of `-`.